testeng/jobs/upgradePythonRequirements.groovy

```diff
@@ -536,6 +536,18 @@ Map registrar = [
     alwaysNotify: true
 ]
 
+Map repoHealthData = [
+    org: 'edx',
+    repoName: 'repo-health-data',
+    targetBranch: "master",
+    pythonVersion: '3.8',
+    cronValue: cronOffHoursBusinessWeekdayTwiceMonthlyEven,
+    githubUserReviewers: [],
+    githubTeamReviewers: [],  // Reviewer mention unnecessary due to Master's OpsGenie alert.
+    emails: ['archbom@edx.org'],
```

Comment:
@jmbowman who should respond to this upgrade job?

Reply:
Should be arbi-bom both for email and GitHub review tagging, they already handle the other repo health repositories.